frontend/src/components/applications/SchoolApplicationForm.vue

```diff
@@ -59,32 +68,51 @@
               <v-divider></v-divider>
               <v-window v-model="tab">
                 <v-window-item
-                  v-for="t in tabContent"
+                  v-for="t in TAB_CONTENT"
                   :key="t.tab"
                   :value="t.tab"
+                  :transition="false"
                 >
                   <keep-alive>
                     <component
                       :is="t.component"
                       :formData="formData"
-                      :draftCode="draftCode"
+                      :draftCode="APP_STATUS_CODES.draft"
```

Comment:
why there is no dynamic binding here?

Reply:
Discussed and prop has been removed as only one component relied on it, so I imported the constant directly into the component.